test/test_codec.py

```diff
@@ -205,3 +205,19 @@ def test_import_export_additional_frame_info(self):
             with open(out_file_name, "r") as file:
                 data = file.read()
             self.assertIn("UserFrameAttr", data)
+
+    def test_export_jsonall(self):
+        """Check the jsonAll doesn't raise and export some additional field."""
+        db_path = os.path.join(
+            os.path.dirname(__file__), "..", "test", "test.dbc")
+        dbs = formats.loadp(db_path)
+        tmp_dir = tempfile.mkdtemp()
+        out_file_name = tmp_dir + "/output.json"
+        formats.dumpp(dbs, out_file_name, jsonAll=True)
```

Comment:
I should know this for sure...  but I _think_ we could just dump it to a string?  :[ I really ought to know this offhand.  If nothing else, I'm pretty sure we could dump it into an `io.BytesIO`.

Reply:
BytesIO applied